optimism.tokenlist.json

```diff
@@ -6,7 +6,7 @@
     "layer2",
     "infrastructure"
   ],
-  "timestamp": "2023-04-14T15:43:37.999Z",
+  "timestamp": "2023-04-17T17:54:43.560Z",
```

Comment:
Unrelated to this pr and also not important but the non hermetic nature of timestamps in build outputs is a nit of mine

Reply:
agreed